src/pages/EditRequestPage.js

```diff
@@ -93,7 +89,8 @@ function EditRequestPage({report, route, parentReport, policyCategories, policyT
     const tagListName = PolicyUtils.getTagListName(policyTags);
 
     // A flag for verifying that the current report is a sub-report of a workspace chat
-    const isPolicyExpenseChat = useMemo(() => ReportUtils.isPolicyExpenseChat(ReportUtils.getRootParentReport(report)), [report]);
+    const policy = ReportUtils.getPolicy(report.policyID || '');
+    const isPolicyExpenseChat = policy && policy.type !== CONST.POLICY.TYPE.PERSONAL;
```

Comment:
`getPolicy` [can return](https://github.com/Expensify/App/blob/8cb48e468ce3c09ada2d58aaae5354ec206104f9/src/libs/ReportUtils.ts#L433-L437) {} and this not equals condition can return `true` in such cases. This !== check kinda appears brittle. Can this check somehow be done with an === check?

Reply:
Yeah you're right, will update!

Reply:
I changed this line because for some reason sometimes the parent report is not available and even though we're inside a policy expense `isPolicyExpenseChat` turns out to be false

Reply:
Or maybe it's the `useMemo` that was causing the bug..